lib/mj-single.js

```diff
@@ -543,7 +543,10 @@ function GetSpeech(result) {
   if (!data.mml) return;
   var jax = MathJax.Hub.getAllJax()[0];
   jax.root.alttext = result.speakText;
-  if (jax.root.attrNames) {jax.root.attrNames.push("alttext")}
+  var attrNames = jax.root.attrNames;
+  if (attrNames & (attrNames.indexOf("alttext") == -1)) {
```

Comment:
This should be `&&` not just `&` (which is a bitwise operation).  Then you will not need the extra parentheses around the equality check.  The equality should be `===` rather than `==`, since no type-coersion is needed.